python/py-setuptools_scm_git_archive/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-setuptools_scm_git_archive
+version             1.0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+master_sites        pypi:s/setuptools_scm_git_archive
+distname            setuptools_scm_git_archive-${version}
+
+python.versions     27 34 35 36 37
+
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         Plugin for setuptools_scm that add support for git archives.
+long_description    ${description}
+
+homepage            https://github.com/Changaco/setuptools_scm_git_archive
+
+checksums           rmd160  665c374d66941d878284e72dcd40125fb88debd3 \
+                    sha256  52425f905518247c685fc64c5fdba6e1e74443c8562e141c8de56059be0e31da \
+                    size    3050
+
+if {${name} ne ${subport}} {
+    depends_lib-append  port:py${python.version}-setuptools_scm
```

Comment:
We seem to be missing the usual bit that turns off livecheck off for the subports. (I wonder if that shouldn't just be part of the portgroup?)

Reply:
Sorry my fault... I added it back